stable/external-dns/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Deploys external-dns into a kubernetes cluster
+name: external-dns
+version: 0.0.1
+appVersion: 0.3.0
```

Comment:
Let's go with version `0.4`. As the final version isn't released yet you can use `v0.4.0-alpha.1` for that matter. I'll make sure it gets released before the chart is done.

We changed some flags in `v0.4` and then we don't have to change them again.